### PR TITLE
Create collection with example queries

### PR DESCRIPTION
**Pull request by edongashi**

### CODE REVIEW

File: src/AdminTab/TableList.tsx [hunk 1]
```diff
@@ -41,6 +41,15 @@ const TableDropdown: FunctionComponent<TableDropdownProps> = ({ table, onOpenMet
       >
         {t('New SQL query')}
       </Menu.Item>
+      <Menu.Item
+        icon={<ConsoleSqlOutlined />}
+        key={`${table.name}-examples`}
+        onClick={async () => {
+          await getTableExamples(table).result;
```
Comment (edongashi) [inline src/AdminTab/TableList.tsx:48]: Gives the collection ID that we use to send the user to metabase.

File: src/AdminTab/TableList.tsx [hunk 2]
```diff
@@ -41,6 +41,15 @@ const TableDropdown: FunctionComponent<TableDropdownProps> = ({ table, onOpenMet
       >
         {t('New SQL query')}
       </Menu.Item>
+      <Menu.Item
+        icon={<ConsoleSqlOutlined />}
```
Comment (pdobacz) [inline src/AdminTab/TableList.tsx:45]: Not necessarily now, but let's remember to change the icon (we can use the chart one we had before)

File: src/main/metabase/api.ts [hunk 3]
```diff
@@ -286,3 +308,141 @@ export async function syncMetabaseSchema(): Promise<void> {
...
+    },
+    display: query.display,
+    description: null,
+    visualization_settings: query.visualizationSettings,
+    parameters: [],
+    collection_id: collectionId,
+    result_metadata: null,
+  });
+
+  console.log(response);
```
Comment (pdobacz) [inline src/main/metabase/api.ts:377]: maybe log a `"Response from POST /api/card", response` so that it is later more easier to find

**Review by pdobacz — APPROVED** [review]

File: src/AdminTab/TableList.tsx (hunk 2 above)
Comment (edongashi) [reply src/AdminTab/TableList.tsx:45]: Ah yes, lazy copy and paste to just get something running.

File: src/main/metabase/api.ts (hunk 3 above)
Comment (edongashi) [reply src/main/metabase/api.ts:377]: This should not be here! Forgot to remove it. Will force push.

**Review by pdobacz — APPROVED** [review]

**Pull request merged**